cpp/include/raft/random/rmat_rectangular_generator.cuh

```diff
@@ -65,6 +68,58 @@ namespace raft::random {
  * @{
  */
 template <typename IdxT, typename ProbT>
+void rmat_rectangular_gen(const raft::handle_t& handle,
+                          raft::random::RngState& r,
+                          raft::device_vector_view<const ProbT, IdxT> theta,
+                          raft::device_vector_view<IdxT, IdxT> out,
+                          raft::device_vector_view<IdxT, IdxT> out_src,
+                          raft::device_vector_view<IdxT, IdxT> out_dst,
+                          IdxT r_scale,
+                          IdxT c_scale,
+                          IdxT n_edges)
```

Comment:
Do we still need n_edges to be explicitly passed in? Can this be inferred from the sizes of the 3 output arrays?

Reply:
I've inferred `n_edges` from the mdspan parameters and removed the explicit `n_edges` parameter of both overloads; thanks!